docs/toml.md

```diff
@@ -1213,9 +1213,13 @@ Træfik can be configured to use Kubernetes Ingress as a backend configuration:
 #
 # namespaces = ["default", "production"]
 
-# See: http://kubernetes.io/docs/user-guide/labels/#list-and-watch-filtering
-# labelselector = "A and not B"
+# Ingress label selector to identify Ingress objects that should be processed.
+# See https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/ for details.
```

Comment:
This link should be updated to https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#label-selectors